types/mongoose/index.d.ts

```diff
@@ -2701,6 +2701,8 @@ declare module "mongoose" {
      * Triggers the save() hook.
      */
     create(docs: any[], callback?: (err: any, res: T[]) => void): Promise<T[]>;
+    create(docs: [any], options?: SaveOptions, callback?: (err: any, res: T[]) => void): Promise<T[]>;
```

Comment:
`[any]` means "an array with one element". This is probably not what you meant, and it is redundant with the other declaration of `any[]`.

Reply:
true that, thx!